meilidb-http/src/routes/index.rs

```diff
@@ -184,12 +397,18 @@ pub async fn delete_index(ctx: Context<Data>) -> SResult<StatusCode> {
     }
 }
 
-pub fn index_update_callback(index_name: &str, data: &Data, _status: ProcessedUpdateResult) {
+pub fn index_update_callback(index_uid: &str, data: &Data, _status: ProcessedUpdateResult) {
     let env = &data.db.env;
     let mut writer = env.write_txn().unwrap();
 
-    data.compute_stats(&mut writer, &index_name).unwrap();
-    data.set_last_update(&mut writer, &index_name).unwrap();
+    data.compute_stats(&mut writer, &index_uid).unwrap();
+    data.set_last_update(&mut writer).unwrap();
+
+    if let Some(index) = data.db.open_index(&index_uid) {
+        if let Err(e) = index.main.put_updated_at(&mut writer) {
+            error!("Impossible to update updated_at; {}", e)
+        }
+    }
```

Comment:
In the other functions (i.e. `compute_stats` and `set_last_update`) checking if the index exist is done inside.
In another PR (#313) I am checking if the index exist in another manner, to avoid throwing errors.
I would prefer that you do not check that in here.

Reply:
And would it be possible to you to handle the other errors to avoid the update thread to panic?
Or would you prefer to let me do this in another PR? :)

Reply:
You must check that the update passed or not, and call those function only if it passed.